docs/sources/errata.rst

```diff
@@ -9,6 +9,7 @@ Supported content types:
 * RPMs
 * Container images
 * Advisories
+* VMIs
```

Comment:
Can you please change it to `Virtual machine images`?

Just to be consistent with [KojiSource](https://github.com/release-engineering/pushsource/blob/master/docs/sources/koji.rst)

```suggestion
* Virtual machine images
```

Reply:
updated